kuma/demos/tests/test_helpers.py

```diff
@@ -14,9 +12,9 @@ def test_tag_description_no_description(self):
             "title": "Testing tag without description",
         }
         description = tag_description("test_tag")
-        eq_("test_tag", description)
+        self.assertEqual("test_tag", description)
```

Comment:
woops.

compared to `assertEqual`. My secret long term plan is to move us to pytest as it's becoming the defacto standard for testing while nose's big time is in the past.